<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="25,75">
    <item>
     <layout class="QVBoxLayout" name="layoutLeftColumn">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPortofolioValueAmount">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Portfolio Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_portfolio_value">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPortofolioValueAmount_3">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Benchmark Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_benchmark_value">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPortofolioValueAmount_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Unit Count</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_unit_count">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPortofolioValueAmount_4">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Instruction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="text_edit_instruction">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_start">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layoutRightColumn" stretch="1,1,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layoutGraphPrice"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutGraphDelta"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Price/Unit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_price_unit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_time">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxNextDecision">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Next Adjustment on Allocation in the Stock Market</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2,2,2,2,2,2,2,2,2,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>20</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>30</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>40</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>50</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>60</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>70</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>80</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>90</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>100</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="indent">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QSlider" name="slider_asset">
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>0</number>
             </property>
             <property name="value">
              <number>5000</number>
             </property>
             <property name="sliderPosition">
              <number>5000</number>
             </property>
             <property name="tracking">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBothSides</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Assets in Stock Market</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_asset_value">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_asset_percentage">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Cash in the Bank       </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_cash_value">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_cash_percentage">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="mouseTracking">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
